src/ol_infrastructure/applications/edxapp/Pulumi.applications.edxapp.mitxonline.Production.yaml

```diff
@@ -45,6 +45,8 @@ config:
   edxapp:mail_domain: edxapp-mail.mitxonline.mit.edu
   edxapp:proctortrack_url: https://testing.verificient.com
   edxapp:sender_email_address: mitxonline-support@mit.edu
+  edxapp:email_use_course_id_from_for_bulk: "True"
+  edxapp:bulk_email_default_from_email: no-reply@mit.edu
```

Comment:
@pdpinch , could you please confirm if the email no-reply@mit.edu is appropriate for our use? It seems to be more general rather than specific to mitxonline. Previously, we used mitxonline-support@mit.edu on production.

Reply:
Good point. We should use no-reply@mail.mitxonline.mit.edu

Reply:
Thank you Peter for the suggestion. I have updated the production bulk email address to no-reply@mail.mitxonline.mit.edu in the pull request. 

It would require an approving review to merge if everything looks good.